src/test/java/io/tiledb/spark/TileDBDataSourceReadTest.java

```diff
@@ -83,6 +85,8 @@ public void sparseHeterogeneousArrayCreate(List<Dimension> dimensions) throws Ti
     schema.addAttribute(a1);
 
     Array.create(SPARSE_ARRAY_URI, schema);
+    domain.close();
```

Comment:
What do `domain.close` and `schema.close` do?

Reply:
See my answer below